src/invoices/InvoiceDetails/InvoiceLines/InvoiceLines.js

```diff
@@ -66,8 +70,22 @@ class InvoiceLines extends Component {
   }
 
   render() {
-    const { resources } = this.props;
+    const { resources, currency } = this.props;
     const invoiceLinesItems = get(resources, 'invoiceLines.records.0.invoiceLines', []);
+    const resultsFormatter = {
+      adjustmentsTotal: line => (
+        <AmountWithCurrencyField
+          amount={get(line, 'adjustmentsTotal', 0)}
```

Comment:
why do you use `get()` here?

Reply:
I thought about not required field but checked again invoice line schema - `adjustmentsTotal` and `Total` are always calculated on the back-end. So `get( )` doesn't make sense here

Reply:
1. the thing is that we suppose and shouldn't check `line` for `null` since if it is - we have a big issue on back-end and shouldn't hide it. It's better to fail front-end ASAP and fix back-end (possible it's sample data). I think it's safe to do because it will be raised at a sanity check after deploy. 
2. I don't think we should specify default `0` everywhere, it's better to handle this case inside `AmountWithCurrencyField` for consistancy reasons.